stacker/providers/aws.py

```diff
@@ -5,10 +5,23 @@
 
 from . import exceptions
 from .base import BaseProvider
+from ..util import retry_with_backoff
 
 logger = logging.getLogger(__name__)
 
 
+def retry_cf_throttling(fn, attempts=3, args=None, kwargs=None):
```

Comment:
I like making it explicit - that way there's no concern about stomping on function arguments with arguments from retry_with_backoff.